bn-models/src/models/_lookups/viability.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseLookup } from '../_base';
```

Comment:
This is Maturity, but the filename is viability.ts (I think it should be maturity.ts. Be sure to update index.ts as well)

Reply:
Oh wait, we already have Maturity. 
I think this file needs to be implemented.